src/ActiveMQ/Providers/ActiveMQTriggerServiceOperationProvider.cs

```diff
@@ -0,0 +1,388 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Workflows.ServiceProvider.Extensions.ActiveMQ
+{
+    using Apache.NMS;
+    using Apache.NMS.AMQP;
+    using Microsoft.Azure.Workflows.ServiceProviders.Abstractions;
+    using Microsoft.Azure.Workflows.ServiceProviders.WebJobs.Abstractions.Providers;
+    using Microsoft.WindowsAzure.ResourceStack.Common.Collections;
+    using Microsoft.WindowsAzure.ResourceStack.Common.Extensions;
+    using Microsoft.WindowsAzure.ResourceStack.Common.Swagger.Entities;
+    using Newtonsoft.Json.Linq;
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// This is the service operation provider class where you define all the operations and apis.
+    /// </summary>
+    [ServiceOperationsProvider(Id = ServiceId, Name = ServiceName)]
+    public class ActiveMQTriggerServiceOperationProvider : IServiceOperationsTriggerProvider
+    {
+        /// <summary>
+        /// Amount of time to wait to receive new ActiveMQ message.
+        /// </summary>
+        private readonly TimeSpan messageReceiveTimeout = new TimeSpan(0, 0, 0, 1);
+
+        /// <summary>
+        /// The service name.
+        /// </summary>
+        public const string ServiceName = "activemq";
+
+        /// <summary>
+        /// The service id.
+        /// </summary>
+        public const string ServiceId = "/serviceProviders/activemq";
+
+        /// <summary>
+        /// Gets or sets service Operations.
+        /// </summary>
+        private readonly List<ServiceOperation> serviceOperationsList;
+
+        /// <summary>
+        /// The set of all API Operations.
+        /// </summary>
+        private readonly InsensitiveDictionary<ServiceOperation> apiOperationsList;
+
+        /// <summary>
+        /// Get ReceiveMessagesTrigger Service Operation.
+        /// </summary>
+        public readonly ServiceOperation ReceiveMessagesTrigger = new ServiceOperation
+        {
+            Name = "ActiveMQ_ReceiveMessages",
+            Id = "ActiveMQ Receive Messages",
+            Type = "ActiveMQ ReceiveMessages",
+            Properties = new ServiceOperationProperties
+            {
+                Api = ActiveMQTriggerApi.GetFlattenedApi(),
+                Summary = "ActiveMQ Receive Messages",
+                Description = "ActiveMQ Receive Messages",
+                Visibility = Visibility.Important,
+                OperationType = OperationType.ServiceProvider,
+                BrandColor = 0x1C3A56,
```

Comment:
Two different brand Colors are used in this file. Is that correct?

Reply:
fixed

Reply:
@praveensri , can you review please ?